docs/GettingStarted.md

```diff
@@ -2194,6 +2176,25 @@ You can configure sampling rule that allow you keep spans despite their respecti
 
 [//]: # (TODO: See <Single Span Sampling documentation URL here> for the full documentation on Single Span Sampling.)
 
+#### Application-side sampling
+
+While the Datadog agent can sample traces to reduce bandwidth usage, application-side sampling reduces the performance overhead in the host application.
+
+**Application-side sampling drops traces as early as possible. This causes the [Ingestion Controls](https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_controls/) page to not receive enough information to report accurate sampling rates. Use only when reducing the tracing overhead is paramount.**
```

Comment:
One pattern I've done in a few places is to invite customers to get in touch with us if they have use cases that require things that are problematic/we don't quite support.

The intention is getting us a bit of signal on how this is getting used in the field, why it's getting used, and even perhaps giving us the chance to fix or improve whatever is leading the customer to need this in some other way.

What do you think of doing the same here? I was thinking of something along the lines of...

> If you needed to enable this, please let us know by opening an issue on (...), so that we can look into improving ddtrace to better fit your use-case.

Reply:
Added!